addons/humanizer/scripts/resources/mhclo.gd

```diff
@@ -106,23 +107,39 @@ func calculate_mhclo_scale(helper_vertex_array: Array) -> Vector3:
 	var mhclo_scale = Vector3.ZERO
 	for axis in ["x","y","z"]:
 		var scale_data = scale_config[axis]
+		if is_zero_approx(scale_data.length):
```

Comment:
scale config is not always part of the mhclo file, this is why the fem_suit was broken in my testing, but scale 1,1,1 was not functional thus 0.1 worked, it should of course be 0.01 for CM to M so I'm not sure why this is the case.